build-system/tasks/presubmit-checks.js

```diff
@@ -647,19 +647,21 @@ var forbiddenTermsSrcInclusive = {
   '\\.scheduleUnlayout\\(': bannedTermsHelpString,
   'getComputedStyle\\(': {
     message: 'Due to various bugs in Firefox, you must use the computedStyle ' +
-        'helper in style.js.',
+    'helper in style.js.',
     whitelist: [
       'src/style.js',
       'dist.3p/current/integration.js',
     ],
   },
   'decodeURIComponent\\(': {
     message: 'decodeURIComponent throws for malformed URL components. Please ' +
-        'use tryDecodeUriComponent from src/url.js',
+    'use tryDecodeUriComponent from src/url.js',
     whitelist: [
       '3p/integration.js',
       'dist.3p/current/integration.js',
       'examples/pwa/pwa.js',
+      'examples/amp-web-push/publisher-files/helper-frame.js',
```

Comment:
/cc @erwinmombay for gulpfile changes.

Cool. But can you explain this a bit more how you see this? Do you want `publisher-files` to be in the GitHub master so that devs can just pick them up as examples? Or do you want them to be deployed somewhere on CDN and instruct devs to simply proxy them?